.pre-commit-config.yaml

```diff
@@ -16,10 +16,10 @@ repos:
         exclude_types:
           - "markdown"
       - id: check-case-conflict
-  - repo: https://github.com/dbt-labs/pre-commit-hooks
-    rev: v0.1.0a1
-    hooks:
-      - id: dbt-core-in-adapters-check
+#  - repo: https://github.com/dbt-labs/pre-commit-hooks
+#    rev: v0.1.0a1
+#    hooks:
+#      - id: dbt-core-in-adapters-check
```

Comment:
Are you planning on re-enabling this check before merging?  Or if not, can you provide more details as to what was failing and why its ok to skip it now.